config/vufind/Koha.ini

```diff
@@ -25,3 +25,19 @@ PROC        = "Processing Center"
 REF         = "Reference Shelf"
 STAFF       = "Staff Office"
 
+; This section translates Koha's internal block types into strings for on-screen
+; display.
+[Blocks]
+;SUSPENSION = "Account Suspended"
+;OVERDUES = "Account Blocked (Overdue Items)"
+;MANUAL = "Account Blocked"
+;DISCHARGE = "Account Blocked for Discharge"
+
+; This section lets you choose whether to display block comments based upon the
+; blocks type (defaults to false, meaning that the name of the block type will
```

Comment:
I expanded this comment to include more information about default behavior. Please correct me if I misinterpreted anything.

Reply:
More than happy with this change, as it'll make it easier for others to understand the default behavior.